Jenkinsfile

```diff
@@ -11,7 +11,7 @@ pipeline {
           }
           steps {
             echo "Installing Insights..."
-            sh 'pip install --user -e .[develop]'
+            sh 'pip install --user -e .[testing]'
```

Comment:
Why not use virtualenv here on RHEL6 if we are using it on RHEL7 and even with your new changes even for docs?

Reply:
@Glutexo I was actually waiting to see what happens with #1342.  I'm also not sure if `virtualenv` in installed on the docker image that is used for the Python 2.6 build.  I do agree that it should be consistent with the other sections.

Reply:
Let’s just try. :)

Not sure about the presence of _virtualenv_ there in the image, but yesterday I tried to create a virtual environment on my RHEL6 box and it installed and ran (almost) just fine. I say almost, because there is a problem with dependencies (see #1373). It looks like the image has the correct versions already installed, but when you create a plain virtual environment, wrong versions are currently picked and the installation would fail.

I’ll make a quick test and if it fails, I‘d keep this as it is for now and unify it after #1373 gets fixed.

Reply:
So just as expected. Creating a virtual environment causes the installation to fail, just as #1373 describes. Let’s just go without venv for now.

https://jenkins-insights-ci.1b13.insights.openshiftapps.com/blue/organizations/jenkins/insights-core/detail/PR-1335/9/pipeline#step-7-log-20